clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,18 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        const mail = settings.contactEmail || null;
+        if (mail !== null) {
+            this.emailHref = `mailto: ${mail}`;
```

Comment:
[RFC6068](https://tools.ietf.org/html/rfc6068) says that there's no space between `mailto:` and the email address,

Also, in my opinion, it'd be cleaner overall to name this local variable `const contactEmail`, and the instance variable `this.contactHref`.